src/network/providers/ethereum.js

```diff
@@ -113,8 +113,9 @@ const getEthers = chain => {
     }
 
     if (!ethersProviders[chain]) {
-        const url = networks[chain].address;
-        ethersProviders[chain] = new ethers.providers.JsonRpcProvider(url);
+        const host = networks[chain].address;
+        const protocol = networks[chain].tls ? 'https' : 'http';
+        ethersProviders[chain] = new ethers.providers.JsonRpcProvider(`${protocol}://${host}`);
```

Comment:
This fixes an actual bug that was introduced when removing protocols from config. I'm including the fix here and not as a hotfix because it only affects ENS, which is pretty much an unused functionality.